cmd/gke-gcloud-auth-plugin/cred_test.go

```diff
@@ -21,19 +21,23 @@ func (*mockTokenSource) Token() (*oauth2.Token, error) {
 	return &oauth2.Token{
 		TokenType:   "bearer",
 		AccessToken: "default_access_token",
-		Expiry:      time.Date(2022, 1, 1, 0, 0, 0, 0, time.UTC),
+		Expiry:      time.Date(2023, 1, 1, 0, 0, 0, 0, time.UTC),
```

Comment:
not sue why this change is being made.. Please dont make it unless absolutely required... Just adds extra lines to the review.